### PR TITLE
feat: fix invert tag for certain raydium markets

### PR DESCRIPTION
**Pull request by yujin512**
1. fix invert tag for certain raydium markets ex: TRUMP
2. ZK can be cross margined

### CODE REVIEW

File: public/configs/someMarketData.json [hunk 1]
```diff
@@ -0,0 +1,1890 @@
...
+      "denom": "adydx",
+      "amount": "10000000000000000000000"
+    },
+    "meta": {
+      "assetName": "Kitten Haimer",
+      "referencePrice": 0.500492,
+      "cmcId": 30948
+    }
+  },
+  "BOOMER": {
```
Comment (nivasan1) [inline public/configs/someMarketData.json:1494]: Looks like this pair has v [low](https://dexscreener.com/solana/31qgknu26ay21vkwgcvau2nzech56f9crktpxtmylyme) liquidity

Comment (nivasan1) [reply public/configs/someMarketData.json:1494]: I suggest we remove

File: public/configs/someMarketData.json [hunk 2]
```diff
@@ -0,0 +1,1890 @@
...
+      "referencePrice": 0.0492261630240356,
+      "cmcId": -1
+    }
+  },
+  "Bazinga": {
+    "title": "Add Bazinga-USD perpetual market",
+    "summary": "Add the x/prices, x/perpetuals and x/clob parameters needed for a Bazinga-USD perpetual market. Create the market in INITIALIZING status and transition it to ACTIVE status after 3600 blocks. Added via the new market widget.",
+    "params": {
+      "id": 664,
+      "ticker": "Bazinga-USD",
```
Comment (nivasan1) [inline public/configs/someMarketData.json]: Upper-case?

**Review by jaredvu — APPROVED** [review]

File: public/configs/someMarketData.json [hunk 3]
```diff
@@ -0,0 +1,1890 @@
...
+      "denom": "adydx",
+      "amount": "10000000000000000000000"
+    },
+    "meta": {
+      "assetName": "NGMI",
+      "referencePrice": 0.1151970509576157,
+      "cmcId": -1
+    }
+  },
+  "Q": {
```
Comment (nivasan1) [inline public/configs/someMarketData.json]: Looks like this pair also has v low liquidity ([dexscreener](https://dexscreener.com/solana/4qpjcwzzyofpeqf1e9x8vkajzuzpcdcbu2makn42bxzc))

Comment (jaredvu) [reply public/configs/someMarketData.json]: Removed

**Pull request merged**